config/accepted-api-changes.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "type": "io.micronaut.nats.reactive.PublishState",
+    "member": "Class io.micronaut.nats.reactive.PublishState",
+    "reason": "PublishState has been removed"
```

Comment:
Small question: if `PublishState` has been removed then shouldn't the version be bumped to 4.0.0?

Reply:
PublishState was only used internally. It was holding the informations for publishing a new message and I replaced it with the nats message itself, holding all the information.
As I started this project, I've missed adding `@Internal` to the class.

Reply:
Right, thanks!